client/my-sites/themes/thanks-modal.jsx

```diff
@@ -167,18 +169,20 @@ const ThanksModal = React.createClass( {
 } );
 
 export default connect(
-	( state, { site } ) => {
-		const currentThemeId = site && getActiveTheme( state, site.ID );
-		const currentTheme = currentThemeId && getCanonicalTheme( state, site.ID, currentThemeId );
+	( state ) => {
+		const siteId = getSelectedSiteId( state );
+		const siteUrl = get( getSite( state, siteId ), 'URL' );
+		const currentThemeId = getActiveTheme( state, siteId );
+		const currentTheme = currentThemeId && getCanonicalTheme( state, siteId, currentThemeId );
 
 		return {
 			currentTheme,
-			detailsUrl: site && getThemeDetailsUrl( state, currentThemeId, site.ID ),
-			customizeUrl: site && getThemeCustomizeUrl( state, currentThemeId, site.ID ),
-			forumUrl: site && getThemeForumUrl( state, currentThemeId, site.ID ),
-			visitSiteUrl: site && site.URL + ( isJetpackSite( state, site.ID ) ? '' : '?next=customize' ),
-			isActivating: !! ( site && isActivatingTheme( state, site.ID ) ),
-			hasActivated: !! ( site && hasActivatedTheme( state, site.ID ) ),
+			detailsUrl: getThemeDetailsUrl( state, currentThemeId, siteId ),
+			customizeUrl: getThemeCustomizeUrl( state, currentThemeId, siteId ),
+			forumUrl: getThemeForumUrl( state, currentThemeId, siteId ),
+			visitSiteUrl: siteUrl + ( isJetpackSite( state, siteId ) ? '' : '?next=customize' ),
+			isActivating: !! ( isActivatingTheme( state, siteId ) ),
+			hasActivated: !! ( hasActivatedTheme( state, siteId ) ),
 			isThemeWpcom: isWpcomTheme( state, currentThemeId )
```

Comment:
Seems like `siteId` should be returned here, since it is used in `onCloseModal()`. `propTypes` also need updating.